sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -77,6 +86,15 @@ static const struct snd_soc_acpi_adr_device rt1308_1_single_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device workaround_rt1308_1_single_adr[] = {
+	{
+		.adr = 0x000110025D070100,
+		.num_endpoints = 1,
```

Comment:
we need  		.adr_override = 0x000120025D130800, /* same as rt1308_1_single_adr */ ?

Reply:
yes, I didn't touch tgl for now, this is needed as well.

Reply:
fixed now